spec/models/user_import_spec.rb

```diff
@@ -28,7 +28,7 @@
         birth_year: nil, gender: nil, gender_custom: nil, email: "ch@lo.com", nationality: nil)
       expect_user_attribs(3,
         login: "flim.flo", name: "Flim Flo", phone: "+123456789", phone2: nil,
-        birth_year: 1989, email: "f@fl.com", nationality: "ZZ")
+        birth_year: 1989, email: "f@fl.com", nationality: nil)
```

Comment:
"ZZ" used to be the default when it couldn't identify the country. Nil seems like a more logical default, and I doubt this would break anyone's workflow...

Reply:
Did the Zanzibar Independence Party lodge a complaint?